analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -0,0 +1,36 @@
+﻿/*
```

Comment:
The file lives next to `LocksReleasedAllPaths.cs`, which is the other existing rule in "AllRules".